src/components/cookie-consent/cookie-consent.scss

```diff
@@ -108,50 +101,21 @@
 }
 
 .a-switch {
-  justify-content: space-between;
+  display: flex !important;
+  align-items: center;
+
+  button {
+    width: 50px;
+  }
 
 	.a-switch__label {
 		flex: auto;
 	}
 }
 
-// Flexboxgrid classes
-.row {
-	box-sizing: border-box;
-	display: flex;
-	flex: 0 1 auto;
-	flex-direction: row;
-	flex-wrap: wrap;
-	margin-right: -0.5rem;
-	margin-left: -0.5rem;
-}
-
-.col-xs-12,
-.col-xs-6,
-.col-md-8,
-.col-md-4 {
-	box-sizing: border-box;
-	flex: 0 0 auto;
-	padding-right: 0.5rem;
-	padding-left: 0.5rem;
-}
-
-.col-xs-12 {
-	flex-basis: 100%;
-	max-width: 100%;
-}
-
-.col-xs-6 {
-	flex-basis: 50%;
-	max-width: 50%;
-}
-
-.col-md-8 {
-	flex-basis: 66.66666667%;
-  max-width: 66.66666667%;
-}
-
-.col-md-4 {
-  flex-basis: 33.33333333%;
-  max-width: 33.33333333%;
+.m-cookie-consent {
+  position: fixed;
+  bottom: 0;
+  z-index: 999;
```

Comment:
Hier hebben we een eigen function voor in SASS: zie regel 158 van https://github.com/a-ui/core_branding_scss/blob/master/src/styles/quarks/_quarks.variables.scss - bv: `layer('overlay');`